### ISSUE TITLE
Add an Openbox desktop profile when there is an active target setup

### ISSUE THREAD

**Issue by cuberhaus** [cuberhaus/dotfiles#21]
## Status
Future work. Do not add placeholder Openbox configuration until there is a machine where it will be used and tested.

## Goal
Provide a complete, reproducible Openbox session rather than tracking an isolated `rc.xml` with undocumented dependencies.

## Prerequisites
Decide and record:
- Target distribution and display manager/session startup path.
- X11 compositor, panel, launcher, notification daemon, wallpaper tool, and policy agent.
- Keyboard layout, monitor handling, terminal, browser, file manager, and screenshot tooling.
- Required keybindings and how they coexist with current i3/XMonad conventions.
- Whether configuration is shared across machines or selected through a host/profile layer.

## Future implementation scope
- Store Openbox configuration in a Stow-managed, platform-appropriate location.
- Declare all runtime packages in the relevant bootstrap profile.
- Include session startup/autostart and required ancillary configuration.
- Avoid duplicating shared shell, terminal, and application configuration.
- Document selection, fallback session, and uninstall/rollback behavior.

## Acceptance criteria
- A fresh supported target can enter a functional Openbox session after bootstrap and Stow setup.
- Essential controls work without manual edits: launch terminal, open launcher, switch/close windows, lock, logout, and take screenshots.
- Autostart dependencies are declared and failures are visible.
- Existing desktop/window-manager sessions remain selectable and unchanged.
- Configuration passes available syntax checks and has a documented manual smoke-test checklist.
- README stops marking Openbox as WIP only after this acceptance criteria is met.

**Comment by cuberhaus** [comment]
> *This was generated by AI during triage.*

## Triage Notes

**What we have established so far:**

- This is a future enhancement for an Openbox desktop profile.
- No Openbox profile exists; the active window-manager surface covers other environments.
- The issue requires an active target setup before implementation.

**What we still need from you:**

- Target distribution and display-manager or session path.
- Compositor, panel, launcher, notification system, and wallpaper choices.
- Monitor, keyboard, and policy-agent requirements.
- Coexistence rules with the existing profiles.

**Comment by cuberhaus** [comment]
> *This was generated by AI during triage.*

## Triage Notes

**What we've established so far:**

- No active Openbox target setup exists yet.
- A generic profile should not be invented without a real target and reproducible requirements.
- The request remains a future enhancement rather than an implementation task.

**What we still need from you (@reporter):**

- A target distribution and session workflow.
- The compositor, panel, launcher, monitor, and policy-agent requirements to preserve.
